src/main/webapp/resources/js/pages/projects/project-analyses/components/SingleSampleAnalysisOutputs.jsx

```diff
@@ -0,0 +1,182 @@
+        let fileName = splitItems[splitItems.length - 1];
+        return fileName + " (" + record.analysisOutputFileKey + ")";
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.analysisType"),
+      key: "analysisType",
+      dataIndex: ["analysisType", "type"],
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.pipeline"),
+      key: "pipeline",
+      dataIndex: "workflowDescription",
+      render(workflowDescription) {
+        return workflowDescription.name + " " + workflowDescription.version;
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.analysisSubmission"),
+      key: "analysisSubmissionName",
+      dataIndex: "analysisSubmissionName",
+      render(analysisSubmissionName, record) {
+        return (
+          <a
+            href={setBaseUrl(`/analysis/${record.analysisSubmissionId}`)}
+            target="_blank"
+          >
+            {analysisSubmissionName}
+          </a>
+        );
+      },
+    },
+    {
+      title: i18n("SingleSampleAnalysisOutputs.submitter"),
+      key: "submitter",
+      dataIndex: "userFirstName",
+      render(userFirstName, record) {
+        return userFirstName + " " + record.userLastName;
+      },
+    },
+    {
+      ...dateColumnFormat(),
+      title: i18n("SingleSampleAnalysisOutputs.createdDate"),
+      key: "createdDate",
+      dataIndex: "createdDate",
+      sorter: false,
+    },
+  ];
+
+  // Get the selected row ids (AnalysisOutputFile id)
+  let rowSelection;
+  rowSelection = {
+    selectedRowKeys: selected,
+    onChange: (selectedRowKeys) => setSelected(selectedRowKeys),
+    getCheckboxProps: (record) => ({
+      name: record.analysisOutputFileId,
+    }),
+  };
+
+  // Function to filter out outputs by the search term
+  const filterOutputsByTerm = (searchStr) => {
+    if (
+      searchStr.trim() === "" ||
+      searchStr === "undefined" ||
+      searchStr === null
+    ) {
+      setFilteredOutputs(null);
+    } else {
+      searchStr = String(searchStr).toLowerCase();
+      // filePath contains the file name so we use it in the search
+      const outputsContainingSearchValue = outputs.filter(
+        (analysisOutput) =>
+          analysisOutput.sampleName.toLowerCase().includes(searchStr) ||
+          analysisOutput.analysisType.type.toLowerCase().includes(searchStr) ||
+          analysisOutput.filePath.toLowerCase().includes(searchStr) ||
+          analysisOutput.userFirstName.toLowerCase().includes(searchStr) ||
+          analysisOutput.userLastName.toLowerCase().includes(searchStr)
+      );
+      setFilteredOutputs(outputsContainingSearchValue);
+    }
+  };
+
+  return (
+    <>
+      <div
+        style={{
+          paddingBottom: SPACE_MD,
+          display: "flex",
+          justifyContent: "space-between",
+        }}
+      >
+        <Button>
+          <IconDownloadFile style={{ marginRight: SPACE_XS }} />
+          {i18n("SingleSampleAnalysisOutputs.download")}
```

Comment:
`Button` accepts an icon property.  That will automatically handle the spacing and everything

Reply:
Updated in ad1cfed0d01bd31b029c25104beb4659dd3c820e